gef.py

```diff
@@ -5485,7 +5485,7 @@ def set_fs(uc, addr):    return set_msr(uc, FSMSR, addr)
 """
 
 
-        content = """#!/usr/bin/python -i
+        content = """#!/usr/bin/bin/env -S python -i
```

Comment:
Also `/usr/bin/bin/env` is unlikely to be a valid path. Please fix.

Reply:
I use `which()` now.